sql/catalyst/src/test/java/org/apache/spark/sql/catalyst/EnumEncoderSuite.java

```diff
@@ -0,0 +1,32 @@
+package org.apache.spark.sql.catalyst;
```

Comment:
(Copy the copyright header you see in other files)
Or, it looks like Encoders-related tests are otherwise all in JavaDatasetSuite. That may be a better place anyway.